frontend-public/src/components/common/FileUpload.js

```diff
@@ -15,11 +16,15 @@ const propTypes = {
   uploadUrl: PropTypes.string.isRequired,
   maxFileSize: PropTypes.string,
   acceptedFileTypesMap: PropTypes.objectOf(PropTypes.string),
+  onFileLoad: PropTypes.func,
+  chunkSize: PropTypes.number,
 };
 
 const defaultProps = {
   maxFileSize: "100MB",
   acceptedFileTypesMap: {},
+  onFileLoad: () => {},
+  chunkSize: 1048576, //1MB
```

Comment:
Can you make this like 5MB or something? 1MB was cool to see in testing. But if we are getting like 50 MB files, we probably don't want 50 network calls and overwhelm uwsgi and/or nginx and such

Reply:
These aren't async so 1 connection will still be consumed persistently while uploading. Maybe we need metrics on this but I feel like even 5mb could be a lot to ask on rural / spotty networks...